searx/settings.yml

```diff
@@ -1910,6 +1910,28 @@ engines:
     engine: wikicommons
     shortcut: wc
     categories: images
+    search_type: images
+    number_of_results: 10
+
+  - name: wikicommons.videos
+    engine: wikicommons
+    shortcut: wcv
+    categories: videos
+    search_type: videos
+    number_of_results: 10
+
+  - name: wikicommons.audio
+    engine: wikicommons
+    shortcut: wca
+    categories: music
+    search_type: audio
+    number_of_results: 10
+
+  - name: wikicommons.files
+    engine: wikicommons
+    shortcut: wcf
+    categories: files
```

Comment:
Not sure these are really "files" .. like we know from torrents by example.  TBH I don't know a better category for this content.

Reply:
Yes, me neither. I've thought rather long about this too when implementing, the only choices I had were the `files` category or no category (`other`), so I decided that it makes sense to use the `files` category given the fact all results are files actually :)